src/test/resources/logback.xml

```diff
@@ -20,18 +20,18 @@
         </encoder>
     </appender>
 
-    <appender name="FILE_RESP" class="ch.qos.logback.core.FileAppender">
-        <file>target/response-${date}.log</file>
-        <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%-5level] %logger{15} - %msg%n%rEx</pattern>
-        </encoder>
-    </appender>
+<!--    <appender name="FILE_RESP" class="ch.qos.logback.core.FileAppender">-->
```

Comment:
Should we delete these lines instead of commenting?

Reply:
I left it as sometimes we want to debug executions and reading response.log is quite useful.